pkg/operator/ceph/cluster/mon/health.go

```diff
@@ -44,14 +44,37 @@ var (
 type HealthChecker struct {
 	monCluster  *Cluster
 	clusterSpec *cephv1.ClusterSpec
+	interval    time.Duration
 }
 
 // NewHealthChecker creates a new HealthChecker object
 func NewHealthChecker(monCluster *Cluster, clusterSpec *cephv1.ClusterSpec) *HealthChecker {
-	return &HealthChecker{
+	h := &HealthChecker{
 		monCluster:  monCluster,
 		clusterSpec: clusterSpec,
+		interval:    HealthCheckInterval,
 	}
+
+	var checkInterval string
+	checkIntervalCRSetting := clusterSpec.HealthCheck.DaemonHealth.Monitor.Interval
```

Comment:
Why not just assign it directly to `checkInterval`? Then you don't need lines 61-63.
```suggestion
	checkInterval := clusterSpec.HealthCheck.DaemonHealth.Monitor.Interval
```